packages/@react-spectrum/dialog/test/Dialog.test.js

```diff
@@ -69,13 +86,24 @@ describe('Dialog', function () {
     let input2 = getByTestId('input2');
     expect(document.activeElement).toBe(dialog);
 
-    fireEvent.keyDown(document.activeElement, {key: 'Tab'});
+    act(() => {
+      fireEvent.keyDown(document.activeElement, {key: 'Tab'});
```

Comment:
ah, recently determined fireEvent doesn't need to be in an act, did you run into an issue when it wasn't?

Reply:
ah, put it inside the act out of habit, pulling it out seems to work too